packages/web-react/src/components/FileUploader/AttachmentImagePreview.tsx

```diff
@@ -7,21 +7,29 @@ type AttachmentImagePreviewProps = {
   imagePreview: string;
   label: string;
   meta?: FileMetadata;
+  objectFit?: 'contain' | 'cover';
 };
 
-const AttachmentImagePreview = ({ label, imagePreview, meta }: AttachmentImagePreviewProps) => {
-  const { classProps } = useFileUploaderStyleProps({ meta });
-  const { imageCropStyles } = classProps;
+const AttachmentImagePreview = ({ label, imagePreview, meta, objectFit }: AttachmentImagePreviewProps) => {
+  const { classProps } = useFileUploaderStyleProps({ meta, objectFit });
+  const { imageCropStyles, attachmentStyles } = classProps;
 
   return (
     <span className={classProps.attachment.image}>
-      <img src={imagePreview} width={IMAGE_DIMENSION} height={IMAGE_DIMENSION} alt={label} style={imageCropStyles} />
+      <img
+        src={imagePreview}
+        width={IMAGE_DIMENSION}
+        height={IMAGE_DIMENSION}
+        alt={label}
+        style={{ ...imageCropStyles, ...attachmentStyles }}
+      />
     </span>
   );
 };
 
 AttachmentImagePreview.defaultProps = {
   meta: undefined,
+  objectFit: 'contain',
```

Comment:
I think `cover` should be default or at least it should match the default in sass 
```
object-fit: var(--file-uploader-attachment-image-object-fit, cover);
```

Reply:
default changed to cover